### PR TITLE
[12.0] [ADD] Account Payment Return Import SEPA PAIN: missing return reasons

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**
From #226

### CODE REVIEW

**Review by sbidoul — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I'm thinking that we should have `_rec_name = complete_name` or `name_get` with "[Code] Name" for not loosing this codification in UI. What do you think?

**Comment by sbidoul** [comment]
> [Code] Name

Yes good idea. @ThomasBinsfeld can you do it while you are at it?

**Comment by ThomasBinsfeld** [comment]
@pedrobaeza @sbejaoui Voilà

File: account_payment_return/models/payment_return_reason.py [hunk 1]
```diff
@@ -20,3 +20,9 @@ def name_search(self, name, args=None, operator='ilike', limit=100):
         if not recs:
             recs = self.search([('name', operator, name)] + args, limit=limit)
         return recs.name_get()
+
+    @api.multi
+    @api.depends('code', 'name')
```
Comment (pedrobaeza) [inline account_payment_return/models/payment_return_reason.py]: This is not needed, as it's not a compute field

File: account_payment_return/models/payment_return_reason.py [hunk 2]
```diff
@@ -20,3 +20,9 @@ def name_search(self, name, args=None, operator='ilike', limit=100):
         if not recs:
             recs = self.search([('name', operator, name)] + args, limit=limit)
         return recs.name_get()
+
+    @api.multi
```
Comment (pedrobaeza) [inline account_payment_return/models/payment_return_reason.py:24]: This is neither, as it's the default.

File: account_payment_return/models/payment_return_reason.py (hunk 1 above)
Comment (ThomasBinsfeld) [reply account_payment_return/models/payment_return_reason.py]: Right, I removed it.

File: account_payment_return/models/payment_return_reason.py (hunk 2 above)
Comment (ThomasBinsfeld) [reply account_payment_return/models/payment_return_reason.py:24]: Yes, but I prefer keep it explicit.

**Pull request merged**